ansible/roles/real-postfix/templates/postfix/virtual-alias-maps-ldap-reserved.cf.j2

```diff
@@ -0,0 +1,7 @@
+server_host = {{ ldap_server }}
+search_base = ou=VirtualHosts,dc=scripts,dc=mit,dc=edu
+query_filter = (&(objectClass=scriptsVhost)(|(scriptsVhostName=%d)(scriptsVhostAlias=%d))(!(scriptsVhostName=scripts.mit.edu))(scriptsReservedMail=%u))
```

Comment:
It might be helpful to add a comment saying where these attributes come from (the LDAP server adds them dynamically, AIUI).